### ISSUE TITLE
Edition volumes not sorting in numerical order 

### ISSUE THREAD

**Issue by clmahoney** [Princeton-CDH/mep-django#654]
**Describe the bug**
Editions are not sorting in volume order if there is no date present/if the date is the same across all vols. 

**To reproduce**
See Work #3824, The Collected Essays and Papers of George Saintsbury, 1875–1920. There are 4 volumes, all pub 1923. When I add in editions (vol 1, 2, 3, 4), for some reason 4 pops to the top, when it should be at the bottom. 
https://shakespeareandco.princeton.edu/books/saintsbury-collected-essays-papers/
**Expected behavior**
Volumes should be listed in numerical order. A secondary sort field would help resolve this. 

**Screenshots**
![Screen Shot 2020-06-01 at 2 36 10 PM](https://user-images.githubusercontent.com/16673934/83441900-46f5be00-a415-11ea-8bad-ba0de765be67.png)

**Comment by rlskoeser** [comment]
I was mis-remembering, volume is numeric - it's number that (amusingly) is not. I'll add a secondary sort on volume (after date). I think that will resolve this, but you'll be able to confirm when I get it on the test site.

**Comment by rlskoeser** [comment]
@clmahoney I added a secondary sort on volume. Please check if this does what you want by adding similar data to this on the test site. Please check that it doesn't mess up any other multivolume or periodical lists, and that it sorts on date first, then volume.

**Comment by clmahoney** [comment]
- YES, I added in volumes again to our example book, The Collected Essays and Papers of George Saintsbury, 1875–1920, and it sorted correctly in numerical order of volume. When I added in a later date to vol 2, 1924 (while all the others, 1, 3, 4 are 1923), the order switched appropriately to put the pub date last. 
- I checked a couple of multivolumes already in the system and the order didn't seem broken or messed up!
